tensorflow/compiler/tf2tensorrt/segment/segment.cc

```diff
@@ -679,6 +680,96 @@ void AddSegmentForNode(const grappler::GraphProperties* graph_properties,
 
 }  // namespace
 
+string GenerateUnconversionReport(
+    std::map<string, std::map<string, int>>& unconverted_ops_map) {
+  // Fetch whether to print a detailed version of the TF-TRT conversion report.
+  bool show_detailed_conversion_report;
+  TF_CHECK_OK(ReadBoolFromEnvVar("TF_TRT_SHOW_DETAILED_REPORT",
+                                 /*default_value=*/false,
+                                 &show_detailed_conversion_report));
+
+  string unsupported_op_report =
+      StrCat("\n\n", string(80, '#'), "\n",
+             "TensorRT unsupported/unconverted OP Report:");
+  int total_unconverted_ops{0};
+
+  // Copying map data into a vector for descending sorting.
+  std::vector<std::pair<
+      std::string,    // OP Name
+      std::pair<int,  // Total Unconverted of a specific OP
+                std::vector<std::pair<string, int>>  // {Reason, Total for this
+                                                     // reason}
+                >>>
+      unconverted_ops_vec;
+
+  // Populate the vector from the map
+  for (auto& it1 : unconverted_ops_map) {
+    int total_unconverted_op{0};
+    std::vector<std::pair<string, int>> reason_occurances_vect;
+
+    for (auto& it2 : it1.second) {
+      total_unconverted_op += it2.second;
+      reason_occurances_vect.push_back(it2);
+    }
+
+    // Sort in descending number of occurances for the reasons why a given
+    // TensorFlow OP was not converted.
+    std::sort(reason_occurances_vect.begin(), reason_occurances_vect.end(),
+              [](const std::pair<std::string, int>& _a,
+                 const std::pair<std::string, int>& _b) -> bool {
+                return _a.second >= _b.second;
```

Comment:
[here](https://google.github.io/styleguide/cppguide.html#Variable_Names) is the style guide about naming convention.
Please remove leading _ from the variable names. The existing code doesn't use such a prefix either.

Reply:
Fixed